src/App.js

```diff
@@ -1,10 +1,53 @@
-import logo from './logo.svg';
 import './App.css';
+import React from 'react';
+import allRecipes from './recipes.js';
+
+const allRecipes_keys = Object.keys(allRecipes);
+
+class Header extends React.Component {
+  render () {
+    return (
+      <div>
+        <h1>My Recipes</h1>
+      </div>
+    );
+  }
+}
```

Comment:
What do you think about re-writing this as a [functional component](https://reactjs.org/docs/components-and-props.html#function-and-class-components) instead?